src/TextField.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import cx from 'classnames';
+import Label from './Label';
+
+/**
+ * @module TextField
+ */
+class TextField extends React.Component {
+
+	constructor(props) {
+		super(props);
+		this.state = {
+			value: props.value,
+			hasError: props.error,
+			error: props.error
+		};
+		this.handleChange = this.handleChange.bind(this);
+	}
+
+	handleChange(e) {
+		this.setState({ value: e.target.value });
+	}
+
+	render() {
+		const {
+			inputId,
+			name,
+			label,
+			labelClassName,
+			placeholder,
+			formAttrs,
+			className,
+			children,
+			...other
```

Comment:
> ok so its fine to be less explicit?
inside the `render` function, yes.

> should I add some documentation?
Yeah, you could probably mention that attributes passed to the component will be sent to the `<input>` internally. You can also add some `propTypes` to make certain attributes required.

> Is that possible to make it required using `other`?
Yes - `other` is just an alias used inside `render`. The component `propTypes` apply to any `this.props`.

> And are you suggesting to just use `other` for any attributes and get rid of `formAttrs` as well
Yes - feel free to add `propTypes` for anything that you want to require/validate